src/schema/v1/image/proxies/index.ts

```diff
@@ -4,8 +4,6 @@ import config from "config"
 
 const { RESIZING_SERVICE } = config
 
-module.exports = function resizeWith() {
-  if (RESIZING_SERVICE === "embedly")
-    return embedly.apply(null, arguments as any)
-  return gemini.apply(null, arguments as any)
+module.exports = function resizeWith(...args: [any, any, any, any]) {
```

Comment:
interesting! this explicitly says there are 4 arguments being passed?

Reply:
Yep. It's creating a tuple with four types. It should likely be narrowed to be more specific, but the original was a bunch of `any`s and I didn't really want to faff about with it.